provider/image.go

```diff
@@ -14,6 +19,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource"
 	"github.com/pulumi/pulumi/sdk/v3/go/common/resource/plugin"
+	"github.com/ryboe/q"
```

Comment:
Is this temporary or should it be kept in prod?

Reply:
One day I will not forget to remove my debug tools. That day was not this day.